styleguide.config.js

```diff
@@ -23,7 +23,11 @@ module.exports = {
       scripts: [
         {src: 'https://use.typekit.net/bfw8xpa.js'},
         {src: 'src/theme/typekit.js'},
-        {src: 'https://js.stripe.com/v3/'}
+        {src: 'https://js.stripe.com/v3/'},
+        {
+          src: 'https://na-library.playground.klarnaservices.com/lib.js',
```

Comment:
Nice! I was curious how we'd get these components to render in the pattern example library